db/models.py

```diff
@@ -1,4 +1,6 @@
+from attr.validators import max_len
```

Comment:
The import statement for `max_len` from `attr.validators` is incorrect and unnecessary. Django's `CharField` uses `max_length` directly, so this import can be removed.